### DISCUSSION TITLE
Automatically rename result tabs

### DISCUSSION THREAD

**Discussion by McinPlay** [dbeaver/dbeaver#14136] [Q&A]
So, I do have a script with 15 SELECT statements. At this point I need them all run and I do so with Execute Statements in Separate Tabs button.

I receive 15 result sets, all named Results - 1, Results -2 and so on...

Is there any way I can automatically rename them based on a query content? for my current needs, table name is enough, but if any more powerful syntax option is available, I'll take it for the future.

**Comment by niendicker** [comment]
@McinPlay  take a look at this `https://grafana.com/docs/grafana/latest/panels/transformations/types-options/#add-field-from-calculation`

**Reply by McinPlay** [reply]
Possibly overkill... I'm working on Oralce SQL queries right now, will not set up Grafana.
